tools/bmh_test/create_bmh.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -x
+
+# -------------------------------------------------------------------------------------------
+# Description:    This script creates a virtual machine using virt-install,
+#                 adds the virtual machine to VBMC (Virtual BMC) for out-of-band management,
+#                 and applies the configuration to Kubernetes
+#
+# Usage:          make tilt-up -> press button in the right upper corner to create bmhs
+#                 /tools/bmh_test/create_bmh.sh
+# 
+# Prerequecites:  a network with ip address of 192.168.222.1 named bmo-test-network and
+#                 VBMC runing
+# -------------------------------------------------------------------------------------------
+
+set -xe
```

Comment:
```suggestion
set -eux
```
You generally want to use `-eux`, ie. also add failing for undefined variables.
If you'd also use pipes `|` in the script, you'd also want to add `-o pipefail` by default.